lib/components/viewers/stop-viewer.js

```diff
@@ -156,11 +157,23 @@ class StopViewer extends Component {
     zoomToStop(map, stopData)
   }
 
-  _renderHeader = () => {
-    const { enableFavoriteStops, hideBackButton, showUserSettings, stopData } =
-      this.props
+  _renderHeader = (agencyCount) => {
+    const {
+      enableFavoriteStops,
+      hideBackButton,
+      showUserSettings,
+      stopData,
+      transitOperators
+    } = this.props
+
+    // We can use the first route, as this operator will only be used if there is only one operator
+    const stationOperator = transitOperators.find(
```

Comment:
Would love to see a mode icon in the header the same way it is shown in related stops!

Reply:
I think we agreed to save this for the nearby view. I like it though -- great idea!